### ISSUE TITLE
Facility Locator Urgent Care MVP 

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#4193]
## High Level User Story
As a Veteran, I need to be able to find the appropriate urgent care facility for my situation so I can use my benefits for care.

## Hypothesis or Bet
- **If** we determine how to query Vets-API/PPMS for 1) Place of Service (PoS) Pharmacy (01), 2) Urgent Care Walk-In Retail (17) and 3) Urgent Care (20) for In-Network **then** _we will be able to differentiate between the subtypes and return them separately in search results._ **{#4129]**

- **If** we remove "Referral" language from the cards and detail pages for Urgent Care facilities (and possibly pharmacy?) **then** _Veterans will understand the process more clearly._ **[#4075]**

- **If** we add the benefit PDF to the urgent care workflow **then** _Veterans will understand how to access their benefits for care and prescriptions and will avoid out of pocket costs._ **[#3897]** **[#3268]** **[#4075]**

- **If** we help users understand the difference between Walk-In Retail Urgent Care and Urgent Care **then** _they will be more likely to select the most appropriate facility._ **[#3897]** **[#4075]**

- **If** we improve this process for Veterans **then** _we will be able to remove the TriWest link._

## VSA OKR
- Reform Community Care Implementation

## Definition of done
**Functionality**
[ ] FL can query Vets-API/PPMS Place of Service (PoS) Pharmacy (01), Urgent Care Walk-In Retail (17), Urgent Care (20) for In-Network separately.  **[#4129]**
[ ] Veterans can search for urgent care facilities in TriWest **and** Optum networks. **[#3914]**
[ ] Veterans are assisted in understanding the subtype choices for urgent care and are able to determine which facility type meets their current need. 
[ ] Veteran can search by urgent care subtype
[ ] Veteran can differentiate between urgent care subtype in search results
[ ] Facilities returned as results of urgent care search do not reference a need for referral.
[ ] Veterans have easy access to the PDF explaining the urgent care benefit.
[ ] Workflow includes information to help the Veteran understand how to fill an Urgent Care Pharmacy Prescription
 ] MVP design for development based on previous work [#3553], feedback received from user research sessions and the following acceptance criteria.
[ ] Iteration incorporates previous community and urgent care research, Facility Locator discovery research and feedback
[ ] DEPO has been given OCC approval to remove TriWest link.
**Process**
[ ] Design incorporates Review Process VA.gov Design System
[ ] Design QA Review is complete.
[ ] IA Review is complete.
[ ] Content Review is complete. 
[ ] 508 Audit is complete.
[ ] BE Review is complete
[ ] FE Review is complete
[ ] Preliminary ATO  review is complete. 
[ ] Pre-launch ATO is complete. 
[ ] QA test plan has been implemented.
[ ] Launch analytics for measurement 
[ ] Create documentation for Support/Call Center
[ ] Release plan implemented. 

## How to configure this issue
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `tools-be`, `tools-fe`,`content-ia`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `call center`, `research`, `accessibility`, `content`)

**Comment by tlei123** [comment]
@mmiddaugh @brangi , here's the **[Test Plan Summary Report](https://dsvavsp.testrail.io/index.php?/reports/view/47)** for your review.  A PDF export is also attached below.

Defects logged:
#5965 - Copy/styling \[needs some details-reconciliation between the InVision prototypes and the Card-types PDF before any fixing starts]
#5967 - Vet Centers facility-type's service-type dropdown enabled
#5999 - Search button alignment on tablet-screen
#6039 - Results summary subhead updates prematurely.

Feel free to ping me on Slack for further discussion.  Thanks much!  :)


[fl-urgent-care-mvp-qa-rpt-200220.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/4232019/fl-urgent-care-mvp-qa-rpt-200220.pdf)
